src/components/rating/Rating.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import style from './Rating.module.css';
+
+function showRatingStarts(rating) {
+    rating = Math.round(parseFloat(rating));
+
+    switch (rating) {
+        case 0:
+            return "☆☆☆☆☆";
+        case 1:
+            return "★☆☆☆☆";
+        case 2:
+            return "★★☆☆☆";
+        case 3:
+            return "★★★☆☆";
+        case 4:
+            return "★★★★☆";
+        case 5:
+            return "★★★★★";
+        default:
+            return "☆☆☆☆☆";
+    }
+}
+
+function Rating(props) {
+    return (
+        <span className={style.rating}>{showRatingStarts(props.children)}</span>
+    )
+}
+
```

Comment:
Everything fine here, but I guess you could simplify it like this:

```
function Rating(props) {
  let num = Math.round(props.children);

  let star = '';
  switch (num) {
    case 0:
      star = '☆☆☆☆☆';
      break;
    case 1:
      star = '★☆☆☆☆';
      break;
    case 2:
      star = '★★☆☆☆';
      break;
    case 3:
      star = '★★★☆☆';
      break;
    case 4:
      star = '★★★★☆';
      break;
    case 5:
      star = '★★★★★';
      break;
    default:
      break;
  }

  return <span className={style.rating}>{star}</span>;
}
```

Right?!?!